modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,124 @@
+
+==== Roles
+
+A role is a named collection of ACLs which can be assigned to zero or more users. Redpanda provides two default roles, `user` and `admin`:
+
+* `user` - All authorized users have this role by default. There are no ACLs assigned to this role until you add them. This means users have no permissions to interact with any resources you add to your Redpanda cluster until you add relevant permissions to this or another role.
+* `admin` - By default, no users have this role. The default admin role provides full access to all resources in your Redpanda installation.
+
+==== Policy conflicts
+
+You can assign a combination of ACLs and roles to any given principal. ACLs can contain allow permissions, deny permissions, or a combination of both. As a result, users may at times have conflicting policies. If this situation arises, the user is permitted to perform an operation if and only if:
+
+* There does not exist a DENY permission matching the operation.
+* There exists an ALLOW permission matching the operation.
+
+== Managing users and roles
+
+Redpanda allows you to manage your RBAC configurations through both RPK. Examples of each are given here.
+
+=== Create a role
+
+Creating a new role is a two-step process. First you define the role, giving it a unique and descriptive name. Second, you assign one or more ACLs to allow or deny access for the new role. This defines the permissions that are inherited by all users assigned to the role.
+
+Using RPK, administrators create a new role by issuing the command:
+
+[,bash]
+----
+rpk role create [ROLE_NAME]
+----
+
+Once the role is created, administrators assign the role as the allowed principal for one or more ACLs. You can modify existing ACLs or create new ones. For example:
+
+[,bash]
+----
+rpk acl create ... --allow-role [ROLE_NAME]
+----
+
+=== Delete a role
+
+When a role is deleted, Redpanda carries out the following actions automatically:
+
+- All role ACLs are deleted.
+- All users are unassigned from role.
+
+Using RPK, administrations delete a role by issuing the command:
+
+[,bash]
+----
+rpk role delete [ROLE_NAME]
+----
+
+=== Assign a role
+
+Administrators may assign a role to any principal created in their Redpanda instance. A principal is composed of two parts: a type and a name. Redpanda currently supports the `User` type only for principals. All principals, therefore, will have the format of `User:[name]`
+
+Using RPK, administrations assign a role to a principal by issuing the command:
+
+[,bash]
+----
+rpk role assign [ROLE_NAME] [PRINCIPAL]
+----
+
+=== Unassign a role
+
+Administrators may remove a role from a principal without deleting the role. A principal is composed of two parts: a type and a name. Redpanda currently supports the `User` type only for principals. All principals, therefore, will have the format of `User:[name]`
```

Comment:
Administrators may remove a role from a principal without deleting the role. A principal is composed of two parts: a type and a name. Redpanda currently supports the `User` type only for principals. All principals, therefore, will have the format of `User:[name]`
^^ This should come earlier in the doc--please see my earlier comment asking how users and principals differ here...Perhaps put **principal** in the terminology section?

Reply:
@oleiman  and @deniscoady  -- I was using this in part from the ACL documentation which talks more about Principals and Users. @oleiman mentioned that we're shying away from talking about Principals. Do you want me to remove any reference to Principals in the ACL doc and just say, you specify the user with the format `User:[name]`?

Reply:
Mentioned in slack but, for completeness, the important distinction is "(User) Principals" vs "Roles".

The "principal" terminology is not problematic when describing ACLs (unless the ACL applies to a role)